### PR TITLE
Java SDK

### PR DESCRIPTION
**Pull request by bartektartanus**

### CODE REVIEW

File: tests/SDKTest.php [hunk 1]
```diff
@@ -44,6 +44,19 @@ class SDKTest extends TestCase
...
+            ],
+            'envs' => [
+                'java-11' => 'docker run --rm -v $(pwd):/app -w /app/tests/sdks/java --env PUB_CACHE=vendor maven:3.6-jdk-11-slim mvn clean install',
+                'java-14' => 'docker run --rm -v $(pwd):/app -w /app/tests/sdks/java --env PUB_CACHE=vendor maven:3.6-jdk-14-slim mvn clean install',
+            ],
+        ],
+
         'typescript' => [
             'class' => 'Appwrite\SDK\Language\TypeScript',
             'build' => [
```
Comment (eldadfux) [inline tests/SDKTest.php:62]: Need to whitelist 'java' on line 139 so tests will actually run

Comment (bartektartanus) [reply tests/SDKTest.php:62]: Done.

**Comment by eldadfux** [comment]
We had an issue in datacenter where appwrite.io is hosted last night that made the build fail at the start (the PHP tests). I have re-run it and not it's failing at the Java part:
https://travis-ci.org/github/appwrite/sdk-generator/builds/677445970#L547

Seems to not be able to create a working env

**Comment by bartektartanus** [comment]
Tests are finally green, looks like first version is ready to merge. :)

**Comment by eldadfux** [comment]
Amazing work as usual 👍

**Pull request merged**